source/whats-new.txt

```diff
@@ -58,6 +58,24 @@ What's New in 6.0
      import these methods from the driver instead of from
      ``mongodb-client-encryption``.
 
+   - Removed support for strings with a length of 12 from the ``ObjectId``
+     constructor.
+
+   - The ``abortTransaction`` and ``commitTransaction`` methods return
```

Comment:
Suggestion:
Some entries use the "()" suffix to identify methods names. I think it helps make certain less ambiguous and would make sense to make these consistent.